### PR TITLE
Added Filters field in Role entity

### PR DESCRIPTION
**Pull request by jyejare**
$subject

### CODE REVIEW

**Comment by oshtaier** [comment]
add a quick example as we usually do in nailgun PRs like the field is returned in POST or GET queries

**Comment by jyejare** [comment]
For test results please look at -> https://github.com/SatelliteQE/robottelo/pull/5592#issuecomment-346325310

**Comment by jyejare** [comment]
@oshtaier ,

```
In [12]: entities.Role(id=2).read()
Out[12]: nailgun.entities.Role(location=[], filters=[nailgun.entities.Filter(id=40), nailgun.entities.Filter(id=41), nailgun.entities.Filter(id=42), nailgun.entities.Filter(id=43), nailgun.entities.Filter(id=44), nailgun.entities.Filter(id=45), nailgun.entities.Filter(id=46), nailgun.entities.Filter(id=47), nailgun.entities.Filter(id=48), nailgun.entities.Filter(id=49), nailgun.entities.Filter(id=50), nailgun.entities.Filter(id=51), nailgun.entities.Filter(id=52), nailgun.entities.Filter(id=53), nailgun.entities.Filter(id=54), nailgun.entities.Filter(id=55), nailgun.entities.Filter(id=56), nailgun.entities.Filter(id=57), nailgun.entities.Filter(id=58), nailgun.entities.Filter(id=59), nailgun.entities.Filter(id=60), nailgun.entities.Filter(id=61), nailgun.entities.Filter(id=62), nailgun.entities.Filter(id=63), nailgun.entities.Filter(id=64), nailgun.entities.Filter(id=65), nailgun.entities.Filter(id=66), nailgun.entities.Filter(id=67), nailgun.entities.Filter(id=68), nailgun.entities.Filter(id=69), nailgun.entities.Filter(id=70), nailgun.entities.Filter(id=71), nailgun.entities.Filter(id=72), nailgun.entities.Filter(id=73), nailgun.entities.Filter(id=74), nailgun.entities.Filter(id=75), nailgun.entities.Filter(id=76), nailgun.entities.Filter(id=77), nailgun.entities.Filter(id=150), nailgun.entities.Filter(id=151), nailgun.entities.Filter(id=152), nailgun.entities.Filter(id=157), nailgun.entities.Filter(id=158), nailgun.entities.Filter(id=179), nailgun.entities.Filter(id=180), nailgun.entities.Filter(id=181), nailgun.entities.Filter(id=182), nailgun.entities.Filter(id=208), nailgun.entities.Filter(id=218), nailgun.entities.Filter(id=219), nailgun.entities.Filter(id=220), nailgun.entities.Filter(id=221), nailgun.entities.Filter(id=222), nailgun.entities.Filter(id=223), nailgun.entities.Filter(id=224), nailgun.entities.Filter(id=225), nailgun.entities.Filter(id=240), nailgun.entities.Filter(id=260), nailgun.entities.Filter(id=261), nailgun.entities.Filter(id=262), nailgun.entities.Filter(id=263)], organization=[], id=2, name=u'Organization admin')

In [13]: entities.Role(name='jitenRole').create()
Out[13]: nailgun.entities.Role(location=[], filters=[], organization=[], id=55, name=u'jitenRole')
```

File: nailgun/entities.py [hunk 1]
```diff
@@ -5586,6 +5586,7 @@ def __init__(self, server_config=None, **kwargs):
             ),
             'location': entity_fields.OneToManyField(Location),
             'organization': entity_fields.OneToManyField(Organization),
+            'filters': entity_fields.OneToManyField(Filter),
```
Comment (abalakh) [inline nailgun/entities.py]: Oh, btw, please sort fields alphabetically

**Comment by jyejare** [comment]
@abalakh , Sorted alphabetically!

**Pull request merged**